src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -0,0 +1,828 @@
+          <v-col id="sample-resolution-card-right-col" cols="6" class="pt-6 pr-15">
+            <div class="download-link-container ml-1 pt-5 pb-5">
+              <v-icon color="primary" class="mt-n1">mdi-file-pdf-outline</v-icon>
+              <a :href="documentURL" download class="mt-2">
+                {{getCreateResolutionResource.sampleFormSection.downloadDocLabel}}
+              </a>
+            </div>
+          </v-col>
+          <v-col cols="2" class="pt-6"></v-col>
+        </v-row>
+      </v-card>
+    </section>
+
+    <section id="resolution-date-section" v-if="entityFilter(CorpTypeCd.COOP)" class="mt-10">
+      <header id="resolution-date-header">
+        <h2>{{getCreateResolutionResource.resolutionDateSection.header}}</h2>
+      </header>
+      <p class="mt-2">
+        {{getCreateResolutionResource.resolutionDateSection.description}}
+      </p>
+      <div :class="{ 'invalid-section': getShowErrors && !this.isResolutionDateValid }">
+        <v-card flat id="resolution-date-card" class="pt-7 pb-5 pl-6 pr-6">
+          <v-row>
+            <v-col id="resolution-date-card-left-col" cols="2" class="pt-6" >
+              <v-card-title class="resolution-date-vcard-title pl-1 pr-0">
+                <div>Resolution Date</div>
+              </v-card-title>
+            </v-col>
+            <v-col id="resolution-date-card-right-col" cols="10" class="pt-6 pl-6 pr-5">
+              <date-picker
+                ref="resolutionDatePickerRef"
+                title="Resolution Date"
+                nudge-right="40"
+                :initialValue="resolutionDateText"
+                :minDate="resolutionDateMinStr"
+                :maxDate="resolutionDateMaxStr"
+                :inputRules="resolutionDateRules"
+                @emitDateSync="onResolutionDateSync"
+              />
+            </v-col>
+          </v-row>
+        </v-card>
+      </div>
+    </section>
+
+    <section id="resolution-text-section" v-if="entityFilter(CorpTypeCd.COOP)" class="mt-10">
+      <header id="resolution-text-header">
+        <h2>{{getCreateResolutionResource.resolutionTextSection.header}}</h2>
+      </header>
+      <p class="mt-2">
+        {{getCreateResolutionResource.resolutionTextSection.description}}
+      </p>
+      <div :class="{ 'invalid-section': getShowErrors && !this.isResolutionTextValid }">
+        <div>
+          <v-card flat id="resolution-text-card" class="pt-7 pb-5 pl-6 pr-6">
+            <v-row>
+              <v-col id="resolution-text-card-left-col" cols="2" class="pt-6" >
+                <v-card-title class="resolution-text-vcard-title pl-1 pr-0">
+                  <div>{{getCreateResolutionResource.resolutionTextSection.textLabel}}</div>
+                </v-card-title>
+              </v-col>
+              <v-col id="resolution-text-card-right-col" cols="10" class="pt-6 pl-6 pr-5">
+                <v-form ref="resolutionTextFormRef">
+                  <v-textarea ref="resolutionTextRef"
```

Comment:
There is an issue I'm having with the autogrow funcitonality in this `v-textarea`.  It autogrows fine when typing inputting text but on resumin of a draft it doesn't autogrow properly.  Instead I believe it just grows to the height that is defined by the `rows` prop value.  I tried to wrap it with a `<v-lazy>` and that works but introduces other issues as it doesn't load the component until a user actually views the step that the component.

Reply:
Can you tell how many text rows are in this component? Could you use that to resize it on draft resume?

If all else fails, check with UX, but IMO you can size this component "tall enough" initially (eg, 3-4 rows) and then don't worry about draft resume...

Reply:
I've not! Looking at this now though. 
The issue is that the extended field is hidden when returning from a draft where the text exceeds the initial area?

Reply:
Definitely agree with Severin, it's probably not worth digging into too too much, but it could be that components size is reactive based on the `value:` prop. You could try applying the model to the value, if there is an initial value, that could potentially be picked up by the component.
Just spitballing, but that might be the prop it watches, not sure!

Reply:
i'll try fiddling with resize a bit and talk to UX if i don't get anywhere with it.

Reply:
Also, the props tab spacing seems a little different than the spacing used elsewhere in this component.

Reply:
See `date-picker` line 112

Reply:
Feel free to resolve this convo at your discretion!